.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Please perform a [cursory search](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+repo%3Aubccm%2Fubc-def) before reporting an issue.**
+
+**Describe the bug in detail**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```

Comment:
Maybe we should have OS Version and Browser version as separate things?

Reply:
It's asking for both OS and Browser in the single field, but we can separate them out if it's not clear.

Reply:
"OS", "Browser", "Version" are the three headings.

Version of what? That's why I think we should be explicit. And from experience, the same browser version on different OS Versions can cause bugs that don't appear on both. I'd prefer

"OS" i.e. Windows
"OS Version" i.e. 10
"Browser" i.e. Chrome
"Browser Version" i.e. 74

Reply:
Made those fields more explicit.